challenge.py

```diff
@@ -0,0 +1,156 @@
+import os
+import re
+
+
+List_calls = "calls.txt"
+List_files = "list_files.txt"
+result_stat = "stat.txt"
+
+#set flags to optimise computation files or call if not needed
+flag_file = 1
+flag_calls = 1
```

Comment:
I stand by my first opinion, these flags are not useful since they are statically defined, and unassigned anywhere else. They could be replaced by a `if True` and it would have the same exact behaviour.

Reply:
Yes, I agree in our case it is useless. Already I removed them